src/App.js

```diff
@@ -0,0 +1,35 @@
+import {
+  BrowserRouter, Routes, Route, Link,
+} from 'react-router-dom';
+import './App.css';
+
+import BookForm from './components/BookForm';
+import Categories from './components/Categories';
+
+function App() {
+  return (
+    <>
+
+      <BrowserRouter>
+        <h1>Awesome books</h1>
+        <nav>
+          <ul>
+            <li>
+              <Link to="/">Add of books | </Link>
+            </li>
+            <li>
+              <Link to="/categories"> Categories  </Link>
+            </li>
+          </ul>
+        </nav>
```

Comment:
- [ ] As requested by the last CR the link of the navbar should be in a different component, it is a project requirement. Please 🙏🏽 complete all requirements to move to the next milestone. 👇🏽 

> Create components inside of the /components directory:
> Contains a navigation to link to the 2 routes you created

- **[OPTIONAL]** I think AWSOME BOOKS  was a previous project, please 🙏🏽 correct the title for this BOOKSTORE project.